### PR TITLE
fix(manifest): Update manifest member pages to follow the same format

### PR DESCRIPTION
**Pull request by dipikabh**
## Description

**Issue**: At the moment, the manifest member pages lack consistency: some list values in a [table](https://developer.mozilla.org/en-US/docs/Web/Manifest/shortcuts#shortcut_item_values), while in some others, the "[Values](https://developer.mozilla.org/en-US/docs/Web/Manifest/protocol_handlers#values)" section appears after "Examples" or uses a longer section name like "[launch_handler item values](https://developer.mozilla.org/en-US/docs/Web/Manifest/launch_handler#launch_handler_item_values)".

**Goal**: Before updating the content of individual manifest member pages, I want to align them all so that they have the same sections in the same order and follow the same format.

**Changes in this PR**:
- Removed the "Type" table from the top of the page (this info will get covered in the "Values" section).
- Added a "Syntax" section to all pages.
- Replaced the table formats in the "Values" section with a definition list format, where applicable.
- Renamed sections where needed to ensure that they are called "Values" and "Examples".
- Moved the "Values" section to the correct place on the page, before "Examples", where needed.

**Next steps**: I haven't made any content updates yet (except for adding the Syntax section), so some pages do not have a "Values" section at this point. I'll tackle this in my next iteration.

## Motivation

To standardize the structure and format across manifest member pages

## Related issues and pull requests

- Tracking issue: https://github.com/mdn/mdn/issues/560
- `display` page PR: https://github.com/mdn/content/pull/34386
- `icons` page PR: https://github.com/mdn/content/pull/34725

### CODE REVIEW

**Review by pepelsbey — COMMENTED** [review]
It looks very nice! Much more consistent and readable now 😍

I have a small concern, though.

````md
```json
"screenshots": [
  {
    "form_factor": "narrow" | "wide"
  }
]
```
````

Presenting values in the code examples separated by pipes looks clear if you know how JSON works, how strict it is, etc. But when you’re learning about manifest for the first time, you might just copy-paste the sample and have a hard time figuring out why your manifest doesn’t work. Browsers won’t show you any warnings if your manifest is malformed.

I searched for `true | false` kind of cases, and they’re not very common, apart from XSL docs, [for example](https://developer.mozilla.org/en-US/docs/Web/XSLT/Element/number). We can fix this, too. I’ll bring this up on the next content sync. Or you can do it while I’m away :)

I wonder if it would be better to show some sort of obviously non-fitting placeholder with the value type and list values below? These are pretty common on MDN. Something like:

````md
```json
"screenshots": [
  {
    "form_factor": <string>
  }
]
```

- `"narrow"` some desc
- `"wide"` some desc
````

**Comment by dipikabh** [comment]
Thanks for reviewing, @pepelsbey!

You've highlighted how we approach [Syntax sections](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Syntax_sections) across different technology areas on MDN. :)

Some background on how I went about the syntax on these pages: I wanted to add a "Syntax" section that provides a quick, sort of a scannable, template that lists all available values or property-value pairs, including any keyword values, so readers can get an overview at a glance. For style, I aligned it with the JS and API areas, using camelCase variables in the syntax, rather than following the approach in the CSS area of using examples to demonstrate the syntax.

I see your point about the potential confusion from using the `|` notation. It's a really good and nuanced observation!
You're right, it might lead to unexpected outcomes if copied and used directly. Even though our [guideline](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Syntax_sections#general_style_rules) says that "Syntax sections are not meant to show runnable code.", I would lean towards reducing the confusion if possible by avoiding the `|`. And as you've pointed, it's true that this is not a common practice in our documentation for writing syntax.
(I'm probably trying to cram too much information by spelling out the keywords in the syntax. It would have been easier if I could add comments to the JSON 🙂.)

If we skip the keywords in the syntax, we could use the variable name to suggest that specific keywords are expected as values for the said member/property. Combining this idea with your suggestion, we could use:

````md
```json
"screenshots": [
  {
 
    "form_factor": "keywordValue",
    "platform": "keywordValue"
  
  }
]
```
````
and `booleanValue` instead of `true | false`. The subsequent "Values" section will list and explain the possible keyword values in detail. What do you think about this approach?

I'd considered using placeholders like `"<image>"`, `"<urlString>"`, etc. for values, but again, these kind of uses in Syntax in our docs are rare (as in [`If-None-Match: "<etag_value>"`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match#syntax)).

File: files/en-us/web/manifest/display/index.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ The `display` manifest member is used to specify your preferred display mode for
 ## Syntax
 
 ```json
-"display": "standalone"
+"display": "fullscreen" | "standalone" | "minimal-ui" | "browser"
```
Comment (dipikabh) [inline files/en-us/web/manifest/display/index.md:15]:
```suggestion
"display": "keywordValue"
```

File: files/en-us/web/manifest/orientation/index.md [hunk 2]
```diff
@@ -7,23 +7,20 @@ browser-compat: html.manifest.orientation
...
 > **Note:** `orientation` and/or its specific values might not be supported by a user agent on various display modes because supporting them does not make sense for the particular context.
 
 > [!NOTE]
 > The orientation can be changed at runtime via the [Screen Orientation API](/en-US/docs/Web/API/Screen_Orientation_API).
 
-## Values
+## Syntax
+
+```json
+"orientation": "any" | "natural" | "landscape" | "landscape-primary" | "landscape-secondary" | "portrait" | "portrait-primary" | "portrait-secondary"
```
Comment (dipikabh) [inline files/en-us/web/manifest/orientation/index.md:20]:
```suggestion
"orientation": "keywordValue"
```

File: files/en-us/web/manifest/prefer_related_applications/index.md [hunk 3]
```diff
@@ -9,19 +9,16 @@ browser-compat: html.manifest.prefer_related_applications
...
-</table>
-
 The `prefer_related_applications` member is a boolean value that specifies that applications listed in [`related_applications`](/en-US/docs/Web/Manifest/related_applications) should be preferred over the web application. If the `prefer_related_applications` member is set to `true`, the user agent might suggest installing one of the related applications instead of this web app.
 
 If omitted, `prefer_related_applications` defaults to `false`.
 
+## Syntax
+
+```json
+"prefer_related_applications": true | false
```
Comment (dipikabh) [inline files/en-us/web/manifest/prefer_related_applications/index.md:19]:
```suggestion
"prefer_related_applications": booleanValue
```

File: files/en-us/web/manifest/related_applications/index.md [hunk 4]
```diff
@@ -9,20 +9,34 @@ browser-compat: html.manifest.related_applications
...
 
 > [!NOTE]
 > Developer can specify that the native applications are preferred over the web application by setting `prefer_related_applications` to `true`.
 
+## Syntax
+
+```json
+"related_applications": [
+  {
+    "platform": "platformValue",
```
Comment (dipikabh) [inline files/en-us/web/manifest/related_applications/index.md:22]:
```suggestion
    "platform": "keywordValue",
```

File: files/en-us/web/manifest/screenshots/index.md [hunk 5]
```diff
@@ -9,18 +9,24 @@ browser-compat: html.manifest.screenshots
...
+
+```json
+"screenshots": [
+  {
+    "form_factor": "narrow" | "wide",
+    "label": "screenshotName",
+    "platform": "platformKeyword",
+    "sizes": "imageWidthximageHeight",
+    "src": "imageURL",
+    "type": "image/imageFormat"
```
Comment (dipikabh) [inline files/en-us/web/manifest/screenshots/index.md:24]:
```suggestion
    "form_factor": "keywordValue",
    "label": "screenshotName",
    "platform": "keywordValue",
    "sizes": "widthxheight",
    "src": "imageURL",
    "type": "mimeType"
```

File: files/en-us/web/manifest/serviceworker/index.md [hunk 6]
```diff
@@ -10,57 +10,41 @@ browser-compat: html.manifest.serviceworker
...
 
-## Example
+## Syntax
 
-See [Payment Handler API > Concepts and usage](/en-US/docs/Web/API/Payment_Handler_API#concepts_and_usage).
+```json
+"serviceworker": {
+  "scope": "scopePath",
+  "src": "scriptPath",
+  "use_cache": true | false
```
Comment (dipikabh) [inline files/en-us/web/manifest/serviceworker/index.md:21]:
```suggestion
  "use_cache": booleanValue
```

File: files/en-us/web/manifest/share_target/index.md [hunk 7]
```diff
@@ -9,25 +9,37 @@ browser-compat: html.manifest.share_target
...
 > If you want to share data using the Web Share API, see [Web Share API](/en-US/docs/Web/API/Web_Share_API) and [`navigator.share()`](/en-US/docs/Web/API/Navigator/share).
 
-## Properties
+## Syntax
+
+```json
+"share_target": {
+  "action": "targetURL",
+  "enctype": "encodingType",
+  "method": "GET" | "POST",
```
Comment (dipikabh) [inline files/en-us/web/manifest/share_target/index.md:25]:
```suggestion
  "method": httpMethodKeyword,
```

File: files/en-us/web/manifest/shortcuts/index.md [hunk 8]
```diff
@@ -7,63 +7,52 @@ browser-compat: html.manifest.shortcuts
...
+    "name": "displayName",
+    "short_name": "shortName",
+    "description": "shortcutDescription",
+    "url": "shortcutURL",
+    "icons": [
+      {
+        "src": "iconURL",
+        "sizes": "iconWidthxiconHeight",
+        "type": "image/imageFormat",
+        "purpose": "monochrome" | "maskable | "any"
```
Comment (dipikabh) [inline files/en-us/web/manifest/shortcuts/index.md:26]:
```suggestion
        "sizes": "widthxheight",
        "type": "mimeType",
        "purpose": "keywordValue"
```

**Review by dipikabh — COMMENTED** [review]
@pepelsbey, I've suggested these changes to remove the pipe notation and streamline some other variable names.

File: files/en-us/web/manifest/categories/index.md [hunk 9]
```diff
@@ -7,22 +7,21 @@ browser-compat: html.manifest.categories
...
 
 > **Note:** `categories` are used only as hints for catalogs or stores listing web applications. Like search engines and meta keywords, catalogs and stores are free to ignore them.
 
 > **Note:** `categories` values are lower-cased by the stores and catalogs before processing, so "News" and "news" are treated as the same value. Developers are encouraged to use lower case in the first place.
 
-## Example
+## Syntax
+
+```json
+"categories": ["category1", "category2", "category3"]
```
Comment (dipikabh) [inline files/en-us/web/manifest/categories/index.md:21]:
```suggestion
"categories": ["<category-name>", "<category-name>", ...]
```

File: files/en-us/web/manifest/short_name/index.md [hunk 10]
```diff
@@ -7,17 +7,14 @@ browser-compat: html.manifest.short_name
...
-    </tr>
-  </tbody>
-</table>
-
 The `short_name` member is a string that represents the name of the web application displayed to the user if there is not enough space to display [`name`](/en-US/docs/Web/Manifest/name) (e.g., as a label for an icon on the phone home screen). `short_name` is directionality-capable, which means it can be displayed left-to-right or right-to-left based on the value of the [`dir`](/en-US/docs/Web/Manifest) and [`lang`](/en-US/docs/Web/Manifest) manifest members.
 
+## Syntax
+
+```json
+"name": "shortAppName"
```
Comment (dipikabh) [inline files/en-us/web/manifest/short_name/index.md:15]:
```suggestion
"name": "<short-app-name>"
```

File: files/en-us/web/manifest/shortcuts/index.md [hunk 11]
```diff
@@ -7,63 +7,52 @@ browser-compat: html.manifest.shortcuts
...
+```json
+"shortcuts": [
+  {
+    "name": "displayName",
+    "short_name": "shortName",
+    "description": "shortcutDescription",
+    "url": "shortcutURL",
+    "icons": [
+      {
+        "src": "iconURL",
```
Comment (dipikabh) [inline files/en-us/web/manifest/shortcuts/index.md:23]:
```suggestion
    "name": "<app-name>",
    "short_name": "<short-app-name>",
    "description": "<description>",
    "url": "<url>",
    "icons": [
      {
        "src": "<icon-url>",
```

File: files/en-us/web/manifest/start_url/index.md [hunk 12]
```diff
@@ -7,22 +7,19 @@ browser-compat: html.manifest.start_url
...
 
 A valid `start_url` needs to be same-origin with the document that references the manifest. If `start_url` is unspecified or invalid in any way (such as not a string, not a valid URL, or not a same-origin with the document), the document URL is used.
 
 > [!NOTE]
 > The `start_url` member is purely advisory, and a user agent may ignore it or allow the user to alter it at install time or afterwards.
 
+## Syntax
+
+```json
+"start_url": "startURL"
```
Comment (dipikabh) [inline files/en-us/web/manifest/start_url/index.md:20]:
```suggestion
"start_url": "<url>"
```

**Review by dipikabh — COMMENTED** [review]
Update all placeholder values to follow the same format

File: files/en-us/web/manifest/categories/index.md [hunk 13]
```diff
@@ -7,22 +7,21 @@ browser-compat: html.manifest.categories
...
 
 > **Note:** `categories` are used only as hints for catalogs or stores listing web applications. Like search engines and meta keywords, catalogs and stores are free to ignore them.
 
 > **Note:** `categories` values are lower-cased by the stores and catalogs before processing, so "News" and "news" are treated as the same value. Developers are encouraged to use lower case in the first place.
 
-## Example
+## Syntax
+
+```json
+"categories": ["<category-name>", "<category-name>", ...]
```
Comment (dipikabh) [inline files/en-us/web/manifest/categories/index.md:21]:
```suggestion
"categories": ["<category-name>", "<category-name>"]
```

File: files/en-us/web/manifest/display_override/index.md [hunk 14]
```diff
@@ -9,84 +9,51 @@ browser-compat: html.manifest.display_override
...
-
 The [`display`](/en-US/docs/Web/Manifest/display) member is used to determine the developer's preferred display mode for a website. It follows a process where the browser falls back to the next display mode if the requested one is not supported. In some advanced use cases, this fallback process might not be enough.
 
 The `display_override` member solves this by letting the developer provide a sequence of display modes that the browser will consider before using the `display` member. Its value is an array of display modes that are considered in-order, and the first supported display mode is applied.
 
-## Values
+## Syntax
+
+```json
+"display_override": ["<display-keyword>", "<display-keyword>", ...]
```
Comment (dipikabh) [inline files/en-us/web/manifest/display_override/index.md:19]:
```suggestion
"display_override": ["<display-keyword>", "<display-keyword>"]
```

File: files/en-us/web/manifest/file_handlers/index.md [hunk 15]
```diff
@@ -27,39 +18,36 @@ For example, a PWA can be registered to handle files that match the `text/plain`
...
-## Values
+## Syntax
+
+```json
+"file_handlers": [
+  {
+    "action": "<action-url>",
+    "accept": {
+      "<mime-type>": ["<file-Extension>", ...],
+      ...
```
Comment (dipikabh) [inline files/en-us/web/manifest/file_handlers/index.md:29]:
```suggestion
      "<mime-type>": ["<file-Extension>", "<file-Extension>"],
      "<mime-type>": ["<file-Extension>"]
```

File: files/en-us/web/manifest/launch_handler/index.md [hunk 16]
```diff
@@ -9,46 +9,32 @@ browser-compat: html.manifest.launch_handler
...
-</table>
-
 The `launch_handler` member defines values that control the launch of a web application. Currently it can only contain a single value, `client_mode`, which specifies the context in which the app should be loaded when launched. For example, in an existing web app client containing an instance of the app, or in a new web app client. This leaves scope for more `launch_handler` values to be defined in the future.
 
-## launch_handler item values
+## Syntax
+
+```json
+"launch_handler": {
+  "client_mode": ["<context-keyword>", "<context-keyword>", ...]
```
Comment (dipikabh) [inline files/en-us/web/manifest/launch_handler/index.md:18]:
```suggestion
  "client_mode": ["<context-keyword>", "<context-keyword>"]
```

**Review by dipikabh — COMMENTED** [review]
remove invalid JSON

**Comment by dipikabh** [comment]
Hi Vadim, this is ready for you to take another look. Thanks!

**Comment by dipikabh** [comment]
Moving this to draft until we decide on a format in https://github.com/mdn/content/pull/34725

**Comment by dipikabh** [comment]
Hi @pepelsbey, we reached a consensus in https://github.com/mdn/content/pull/34725#discussion_r1757746394 about the format of "Syntax" blocks. It will contain use cases, similar to the "Syntax" blocks in CSS properties, not placeholder variables. [`icons`](https://developer.mozilla.org/en-US/docs/Web/Manifest/icons) and [`theme_color`](https://developer.mozilla.org/en-US/docs/Web/Manifest/theme_color) manifest pages just landed with these changes to the "Syntax".

What you need to know for reviewing this PR:
- Since adding examples to the "Syntax" block will be a bit more involved now, I've subtracted "Syntax" blocks from this PR.
- I've also removed the files `name`, `short_name`, `start_url` ([35847](https://github.com/mdn/content/pull/35847)), `related_applications`, and `prefer_related_applications` ([35906](https://github.com/mdn/content/pull/35906)) from this PR because they're part of other ongoing PRs.

**Review by pepelsbey — APPROVED** [review]
Looks good :) Thank you!

**Comment by dipikabh** [comment]
Thanks for reviewing, Vadim!

**Pull request merged**